modules/sum-transformers/additional/summary/summary_params_extractor_test.go

```diff
@@ -0,0 +1,100 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package summary
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/graphql-go/graphql/language/ast"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_parseSummaryArguments(t *testing.T) {
+	type args struct {
+		args []*ast.Argument
+	}
+	tests := []struct {
+		name string
+		args args
+		want *Params
+	}{
+		{
+			name: "Should create with no params",
+			args: args{},
+			want: &Params{},
+		},
+		{
+			name: "Should create with all params",
+			args: args{
+				args: []*ast.Argument{
+					createListArg("properties", []string{"prop1", "prop2"}),
+				},
+			},
+			want: &Params{
+				Properties: []string{"prop1", "prop2"},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			p := &SummaryProvider{}
+			if got := p.parseSummaryArguments(tt.args.args); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("parseSummaryArguments() = %v, want %v", got, tt.want)
+			}
+			actual := p.parseSummaryArguments(tt.args.args)
+			assert.Equal(t, tt.want, actual)
+		})
+	}
+}
+
+// func createArg(name string, value string) *ast.Argument {
+// 	n := ast.Name{
+// 		Value: name,
+// 	}
+// 	val := ast.StringValue{
+// 		Kind:  "Kind",
+// 		Value: value,
+// 	}
+// 	arg := ast.Argument{
+// 		Name:  ast.NewName(&n),
+// 		Kind:  "Kind",
+// 		Value: ast.NewStringValue(&val),
+// 	}
+// 	a := ast.NewArgument(&arg)
+// 	return a
+// }
```

Comment:
I think that that his part can be removed

Reply:
Done